build/scripts/build-utils.ps1

```diff
@@ -56,7 +105,7 @@ function Ensure-BasicTool([string]$name, [string]$version) {
     $p = Join-Path (Get-PackagesDir) "$($name).$($version)"
     if (-not (Test-Path $p)) {
         $nuget = Ensure-NuGet
-        Exec { & $nuget install $name -OutputDirectory (Get-PackagesDir) -Version $version }
+        Exec-Block { & $nuget install $name -OutputDirectory (Get-PackagesDir) -Version $version } | Out-Null
```

Comment:
:question: Why are some of these using `Out-Host`, while others are using `Out-Null`?